lib/gooddata/cloud_resources/blobstorage/blobstorage_client.rb

```diff
@@ -0,0 +1,65 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+#
+# Copyright (c) 2021 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'securerandom'
+require 'pathname'
+require "azure/storage/blob"
+
+module GoodData
+  class BlobStorageClient
+    SAS_URL_PATTERN = %r{(^https?:\/\/[^\/]*)\/.*\?(.*)}
+    attr_reader :use_sas
+
+    def initialize(options = {})
+      raise("Data Source needs a client to Blob Storage to be able to get blob file but 'blobStorage_client' is empty.") unless options['blobStorage_client']
+
+      # rubocop:disable Naming/VariableName
+      if options['blobStorage_client']['connectionString'] || options['blobStorage_client']['container']
```

Comment:
Both connectionString and container are mandatory

Reply:
Fixed, connectionString and container are mandatory